cloudbuild.yaml

```diff
@@ -168,24 +168,13 @@ artifacts:
     objects:
         location: gs://open-match-build-artifacts/output/
         paths:
-            - cmd/backend/backend
-            - cmd/frontend/frontend
-            - cmd/mmlogic/mmlogic
-            - cmd/synchronizer/synchronizer
-            - cmd/minimatch/minimatch
```

Comment:
This is ok for now but I may bring back minimatch and certgen later with their Win64 and OSX variants when we have a multi language support story.